package.json

```diff
@@ -54,6 +54,8 @@
   "dependencies": {
     "@types/papaparse": "5.3.2",
     "dayjs": "1.11.0",
+    "js-file-download": "^0.4.12",
```

Comment:
Shouldn't be added here. Should be added under respective apps.

Use `yarn workspace @tbcm/api or web add <package name>`